### PR TITLE
Download the latest subctl

### PR DESCRIPTION
**Pull request by skitt**
This queries the GitHub API to find the latest subctl release, and
downloads that.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

File: package/Dockerfile.dapper-base [hunk 1]
```diff
@@ -29,8 +29,9 @@ ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARC
 # make           | OLM installation
 # findutils      | e2e cleanup (xargs)
 # upx            | binary compression
+# jq             | JSON processing (GitHub API)
 RUN dnf -y distrosync && \
-    dnf -y install --nodocs --setopt=install_weak_deps=False gcc git-core curl moby-engine make golang kubernetes-client findutils upx && \
+    dnf -y install --nodocs --setopt=install_weak_deps=False gcc git-core curl moby-engine make golang kubernetes-client findutils upx jq && \
```
[the remark this answers is not in the document]
Comment (skitt) [reply package/Dockerfile.dapper-base]: This needs Miguel’s fix to handle `dapper-base` changes, I’ve rebased on top of that for this PR.

[the remark this answers is not in the document]
Comment (dfarrell07) [reply package/Dockerfile.dapper-base]: Nice, that fixes it.

**Comment by mangelajo** [comment]
@skitt can you rebase removing the first commit related to the dapper-base part?

**Comment by skitt** [comment]
> @skitt can you rebase removing the first commit related to the dapper-base part?

Done!

**Comment by mangelajo** [comment]
Thanks!!! :-)

**Comment by mangelajo** [comment]
Not waiting for CI to merge, this has been tested before.

**Pull request merged**